### ISSUE TITLE
Alternative (graphic) selection of Heimatbahnhof

### ISSUE THREAD

**Issue by martinheppner** [bahnzumberg/zuugle-suchseite#320]
The usecase is the following: If a new visitor is reaching our searchpage, he doesn't know that he has to select a city first, where he is departing. This concept is unknown to him, why we wouldn't want to force him to select a city, without giving him the information why. This is a new approach to tell him why: We would show a new map, maybe fullscreen, which let's him/her select the city (Heimatbahnhof). Hovering over one city, all hikes are shown on the map, which are reachable from there. The cities are marked clearly and the city, the most people are coming from, is activated on the map by default. 

Short version: Show all Heimatbahnhöfe on the map and draw lines to all reachable tours, Then ask visitor to select Heimatbahnhof.

![Image](https://github.com/bahnzumberg/zuugle-suchseite/assets/105971399/feb4ec2f-65a0-4a4e-929f-764123edde70)
